packages/row/src/css/index.js

```diff
@@ -101,13 +104,18 @@ export default {
     lineHeight: 0,
     cursor: 'pointer',
     border: 'none',
-    color: core.colors.gray02,
     background: 'none',
-    transition: `all ${core.motion.speedNormal}`,
+    transition: `all ${core.motion.speedNormal}`
+  },
+  [`.psds-row__action-bar__button--theme-${themeNames.dark}`]: {
+    color: core.colors.gray02,
 
-    '&:active,&:hover': {
-      color: core.colors.white
-    }
+    '&:active, &:hover': { color: core.colors.white }
+  },
+  [`.psds-row__action-bar__button--theme-${themeNames.light}`]: {
```

Comment:
The theme modifier for selectors has traditionally been something more like:

```
`.psds-row__action-bar__button.psds-theme--${themeNames.light}`
```

Same for line 110 ^.

Reply:
good catch. i'll fix